### DISCUSSION TITLE
Use on android

### DISCUSSION THREAD

**Discussion by sgab31** [gavvvr/obsidian-imgur-plugin#59] [Getting help]
The plugin is compatible with m'y Android obsidian but I dont know how to use or.
I can't copy and paste an image in obsidian.

Please help

**Comment by gavvvr** [comment]
Hi @sgab31 

The mobile support was recently added.
I've tested it with the following use case on both - Android and iOS: 
1. Open your mobile browser -> go to google.com -> find example image
2. Press and hold your finger on the image until context menu appears and click "Copy"
3. Switch to Obsidian -> open the note and hold your finger until context menu appears too. Choose "Paste" context menu option

**Comment by sgab31** [comment]
OK. 

But we can not add an image from our Phone, isn't it ? 



⁣Télécharger BlueMail pour Android ​

Le 23 nov. 2023, 19:45, à 19:45, Kirill Gavrilov ***@***.***> a écrit:

**Reply by gavvvr** [reply]
@sgab31 do you mean the possibility to add an image from the phone's gallery? I haven't tested that, but most probably it does not work because it's not a "pate" event which plugin currently relies on

**Comment by sgab31** [comment]
Yes I mean that. 

It Will be useful because when readin a book I often take photos in order to but thle in obsidian. 

For the moment, I am obliged to put thel non PC before to insert in obsidian

⁣Télécharger BlueMail pour Android ​

Le 24 nov. 2023, 17:14, à 17:14, Kirill Gavrilov ***@***.***> a écrit:

**Reply by gavvvr** [reply]
Got it. I created a #60 feature request from this conversation.

**Comment by sgab31** [comment]
Thanks ! 

Have a Nice day



⁣Télécharger BlueMail pour Android ​

Le 25 nov. 2023, 13:36, à 13:36, Kirill Gavrilov ***@***.***> a écrit:
